README.md

```diff
@@ -1,27 +1,29 @@
-# ejs-loader for webpack
+# ejs-loader for webpack with ECMAScript Modules
 
-[![npm](https://img.shields.io/npm/v/ejs-loader.svg)](https://www.npmjs.com/package/ejs-loader)
-[![Build Status](https://travis-ci.com/difelice/ejs-loader.svg)](https://travis-ci.com/difelice/ejs-loader)
```

Comment:
I think Travis-CI is free to use for OSS, including forks, so we could just change the namespace here to `groveco` and keep the `.travis.yml` basically unchanged... But good work converting to Circle CI!

Reply:
@al-the-x thanks! Since we are maintaining this as an org temporarily (I hope), we probably want to keep CI providers consistent to prevent even more cognitive overhead. I think we should keep the circle config since the changes in this repo will not be contributed directly upstream. https://github.com/difelice/ejs-loader/pull/41 and https://github.com/difelice/ejs-loader/pull/42 are similar to what we have going on here, but took a little bit more work to be backwards compatible (feature flag with option to opt in).